terraform/modules/monitoring/prober.tf

```diff
@@ -0,0 +1,161 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+resource "google_cloud_run_v2_job" "pmap_prober" {
+
+  project = var.project_id
+
+  name = "pmap-prober"
+
+  location = "us-central1"
+
+  template {
+
+    template {
+
+      max_retries = 3
+
+      containers {
+        image = var.pmap_prober_image
+
+        env {
```

Comment:
Can you use `dynamic "env"` here? You can find simimar examples in JVS

Reply:
Updated.